validator/client/validator_attest.go

```diff
@@ -30,10 +30,8 @@ func (v *validator) AttestToBlockHead(ctx context.Context, slot uint64, pk strin
 	defer span.End()
 
 	tpk := hex.EncodeToString(v.keys[pk].PublicKey.Marshal())[:12]
-
-	span.AddAttributes(
```

Comment:
for tracing, maybe its best to leave it as `validator` instead of `pubKey`

Reply:
Would it also make sense to log the full public key for the tracing?

Reply:
I think its truncated form would be fine for tracing